server.mjs

```diff
@@ -22,6 +22,11 @@ const SALT_ROUNDS = 10;
 
 app.use(bodyParser.json());
 app.use(cookieParser());
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
```

Comment:
@vishwa2710 We don't want to allow all origins to make requests to this domain.
Also there is a great cors npm package that we can leverage : https://expressjs.com/en/resources/middleware/cors.html